### ISSUE TITLE
BUG - sev-3 - All - HSP: crisis line SR role currently button (should be link)

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8294]
## What happened?

According to the [home screen personalization Figma](https://www.figma.com/file/q4oTyzyLfYrZVz4CACzVLA/Personalization-Visual-Design---Working---VA-Mobile?type=design&node-id=3053-3306&mode=design&t=26sDG4nZ37ahdsYv-0), the veterans crisis line component should be announced as a link to screen readers.

It's currently announcing as a button (as it did prior to the HSP design/change).

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by TKDickson** [comment]
Now announcing as link, iOS and Android. <img width="361" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOTgzQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5efcfef58c05fcb81070f3c7f8e328beaceafd34/image.png" alt="image.png" />
